layout/_partial/daily_pic.ejs

```diff
@@ -1,5 +1,5 @@
 <!-- Daily pic -->
-<div class="mdl-card daily-pic mdl-cell mdl-cell--8-col index-top-block">
+<div class="mdl-card <% if(theme.card_elevation) { %>mdl-shadow--<%= theme.card_elevation %>dp<% } %> daily-pic mdl-cell mdl-cell--8-col index-top-block">
```

Comment:
I would use `<% if(theme.card_elevation) { %><%= 'mdl-shadow--' + theme.card_elevation + 'dp' %><% } %>` for more readability